nuitka/plugins/PluginBase.py

```diff
@@ -32,6 +32,9 @@
 import sys
 from collections import namedtuple
 
+from ordered_set import OrderedSet
```

Comment:
Cannot do it directly, 3.6 and 3.10 cannot use the same `OrderedSet`, but there is `nuitka.containers.oset` that you can use.